python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -75,14 +75,19 @@ def test_simple_flow(basic_store_no_symbol_list, symbol):
     assert basic_store_no_symbol_list.list_symbols() == basic_store_no_symbol_list.list_versions() == []
 
 
-@pytest.mark.parametrize("special_char", list("$@=;/:+ ,?\\{^}%`[]\"'~#|!-_.()"))
-def test_special_chars(object_version_store, special_char):
+def test_special_chars(object_version_store):
```

Comment:
Why this change?

Reply:
makes the test a bit faster on windows, due to not having to reinit the arctic_client on every option